Models/MetAPI/GraphQLTypes/ForecastType.cs

```diff
@@ -1,8 +1,6 @@
 ﻿using Backend.API.Services;
-using Backend.Models.Base.Metadata.GraphQLTypes;
 using Backend.Models.MetAPI.POCO;
 using Backend.utils.GraphQLTypes;
-using GraphQL.Types;
 
 namespace Backend.Models.MetAPI.GraphQLTypes
```

Comment:
Not changed in your PR, but perhaps assign the nullable field and type field for the last field in the ForecastType?